README.md

```diff
@@ -69,84 +131,50 @@ furyagent
 │   ├── openvpn-client
 │   └── ssh-keys
 ├── backup
-│   ├── etcd
-│   └── master
+│   └── etcd
 └── restore
-    ├── etcd
-    └── master
+    └── etcd
 ```
 
-## Workflow
+## Configure Furyagent
 
-1. Write a [`furyagent.yml`](furyagent.yml)
-2. Generate certificates (by hand at the moment)
-3. `furyagent init -d /path/to/cert/dir --config /path/to/furyagent.yml [etcd|master]` to upload certificates
-4. Then on the nodes: `furyagent configure --config /path/to/furyagent.yml [etcd|master]` to download the certificates to the correct directory specified in the config file
-5. if needed: to backup the state of etcd through `furyagent backup --config /path/to/furyagent.yml etcd`
-6. if needed: to restore the state of etcd, stop etcd, run `furyagent restore --config /path/to/furyagent.yml etcd`, restart etcd
+`furyagent` uses an utility S3 Bucket. To configure access, `furyagent` reads a `Furyagent.yml` config file.
```

Comment:
I think it has to be all under case: `furyagent.yml`

Reply:
default is all lower case: https://github.com/sighupio/furyagent/blob/master/cmd/root.go#L62 @nikever